modules/user-facing-text/main/index.coffee

```diff
@@ -0,0 +1,45 @@
+_ =
+  localisedText: {}
+
+userFacingTextEmitter = new hx.EventEmitter
+
+userFacingText = (stringPath, locales) ->
+  if arguments.length
+    strings = if hx.isString(stringPath)
+      [stringPath]
+    else stringPath
+
+    if locales?
+      newText = tmp = {}
+      for string, i in strings
+        if i is strings.length - 1
+          # Merge the locales to prevent mutation
+          tmp[string] = hx.merge locales
+        else
+          tmp[string] = {}
+          tmp = tmp[string]
+      userFacingTextEmitter.emit('change')
```

Comment:
I'm not sure why you'd need to detect this but I thought it might be a 'nice to have', easy enough to remove if it doesn't make sense.